docker-hub/publish/insights-analytics.md

```diff
@@ -47,84 +48,70 @@ The HTTP API endpoints are available at:
+represents an image pull.
+
+| Data point                    | Value                                             | Description                                                                           | Available from  |
+| ----------------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------- | --------------- |
+| Timestamp                     | `YYYY-MM-DD 00:00:00`                             | Date and time of the request.                                                         | January 1, 2022 |
+| Namespace                     | String                                            | Docker organization (image namespace).                                                | January 1, 2022 |
+| Repository                    | String                                            | Image name.                                                                           | January 1, 2022 |
+| Reference                     | String                                            | Image digest or tag, as requested.                                                    | January 1, 2022 |
+| Digest                        | String                                            | Image digest.                                                                         | January 1, 2022 |
+| Tag (included when available) | String                                            | Tag name. Only available if the request referred to a tag.                            | January 1, 2022 |
+| Action day                    | `YYYY-MM-DD`                                      | The date part of the timestamp.                                                       | January 1, 2022 |
+| HTTP method                   | String                                            | HTTP method used in the request, see [registry API documentation][1] for details.     | January 1, 2022 |
+| Action                        | `pull_by_tag`, `pull_by_digest`, `version_check`  | Request type, see [Action classification rules][2].                                   | January 1, 2022 |
+| Type                          | `business`, `isp`, `hosting`, `education`, `null` | The industry from which the event originates.                                         | January 1, 2022 |
+| Host                          | String                                            | The cloud service provider used in an event.                                          | January 1, 2022 |
+| Country                       | String                                            | Request origin country.                                                               | January 1, 2022 |
+| Domain                        | String                                            | Request origin domain, see [Privacy][3].                                              | October n, 2022 |
```

Comment:
**suggestion (non-blocking):** What do you think of ordering this table chronologically? Maybe in a descending order so you get to see the most recent available data points first

Reply:
yeah I wasn't sure whether to order it like in the CSV or by date. Let's try by date.